app/src/main/res/values/strings.xml

```diff
@@ -14,8 +13,18 @@
     <string name="genres_rock">Rock</string>
     <string name="genres_edm">EDM</string>
     <string name="genres_electronic">Electronic</string>
-
     <string name="image_track">image_track</string>
     <string name="image_more_info">moreinfo</string>
+    <string name="title_add_album">Add Album</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.title_add_album` appears to be unused
 Unused resources make applications larger and slow down builds.